pallets/rmrk-equip/src/functions.rs

```diff
@@ -149,19 +158,19 @@ where
 			!pallet_rmrk_core::Pallet::<T>::nfts(item_collection_id, item_nft_id)
 				.unwrap()
 				.equipped,
-			Error::<T>::AlreadyEquipped
+			Error::<T>::ItemAlreadyEquipped
 		);
 
 		// If the Equippings storage contains the Base/Slot for the Collection+NFT ID, the item is
 		// already equipped
-		let item_is_equipped =
-			Equippings::<T>::get(((equipper_collection_id, equipper_nft_id), base_id, slot_id))
-				.is_some();
-		ensure!(!item_is_equipped, Error::<T>::AlreadyEquipped);
+		ensure!(
+			!Self::slot_is_equipped((equipper_collection_id, equipper_nft_id), base_id, slot_id),
+			Error::<T>::SlotAlreadyEquipped
+		);
 
 		// Item must exist
 		let item_exists =
-			pallet_rmrk_core::Pallet::<T>::nfts(item_collection_id, item_nft_id).is_some();
+			pallet_rmrk_core::Pallet::<T>::nft_exists((item_collection_id, item_nft_id));
```

Comment:
We do what you did on line `166` for this logic here:
`ensure!(pallet_rmrk_core::Pallet::<T>::nft_exists((item_collection_id, item_nft_id)), Error::<T>::ItemDoesntExist);`